src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,9 +3,39 @@
 public class Comment{
     private final String name;
     private final String comment;
+    private final long id;
 
-    public Comment(String name, String comment){
+    public Comment(String name, String comment, long id){
         this.name = name;
         this.comment = comment;
+        this.id = id;
     }
+
+    public String getDisplayName(){
+        return name; 
+    }
+
+    public String getComment(){
+        return comment; 
+    }
+
+    public long getID(){
+        return id; 
+    }
+
+    @Override
+    public boolean equals(Object o) {   
+        if (o == this) { 
+            return true; 
+        } 
```

Comment:
you must implement hashCode() if you implement equals.

Reply:
Swetha will take care of hashcode